tests/test_oic_provider.py

```diff
@@ -829,6 +829,32 @@ def test_userinfo_endpoint_authn(self):
         ident = OpenIDSchema().deserialize(resp.message, "json")
         assert _eq(ident.keys(), ['nickname', 'sub', 'name', 'email'])
 
+    def test_userinfo_endpoint_missing_client(self):
+        self.provider.cdb["unknownclient"] = {
+            "client_secret": "unknownclient",
+            "redirect_uris": [("http://localhost:8087/authz", None)],
+            "post_logout_redirect_uris": [("https://example.com/post_logout", None)],
+            "client_salt": "salted",
+            'response_types': ['code', 'token', 'code id_token', 'none', 'code token', 'id_token']
```

Comment:
Is there any specific reason for using **single** quotes here, as **double** quotes are used in other key pairs?

Reply:
Same applies for other usages

Reply:
No specific. We do have an inconsistent quoting style :(

I will match the quotes inside a test to match.